### PR TITLE
[@types/chrome] Add cloudFileInfo to chrome.fileSystemProvider.notify

### PR DESCRIPTION
**Pull request by BenjaminReich**
The [original PR](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/69240) missed this inclusion in the `chrome.fileSystemProvider.notify` method (which is present on the original CL).  

- [ ] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://crrev.com/c/5343674
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the `package.json`.

### CODE REVIEW

**Review by matcarlson — APPROVED** [review]

**Comment by BenjaminReich** [comment]
Is there anything else I can do here? Seems @matcarlson is a DT maintainer and has pressed "Approved".

**Review by matcarlson — APPROVED** [review]
Approving again - The Typescript bot is supposed to add a comment asking a maintainer or the author to comment "Ready to Merge", then it'll merge the PR. I'm not sure why that didn't happen -- Maybe because I didn't comment? Let's see.

**Comment by matcarlson** [comment]
> Hey @BenjaminReich,
> 
> 😒 Your PR doesn't modify any tests, so it's hard to know what's being fixed, and your changes might regress in the future. Please consider [adding tests](https://github.com/DefinitelyTyped/DefinitelyTyped#user-content-test-editing-an-existing-package) to cover the change you're making. Including tests allows this PR to be merged by yourself and the owners of this module.
> 
> _**This can potentially save days of time for you!**_

@BenjaminReich - This may be the reason it's not merging it immediately -- If you don't add a test it probably requires DefinitelyTyped owners approval. If you aren't able to edit / write a test for this they should approve in the next two weeks -- Usually it's a few days though.

**Review by jakebailey — APPROVED** [review]

**Comment by BenjaminReich** [comment]
Ready to merge

**Pull request merged**
